src/components/captcha/Captcha.tsx

```diff
@@ -1,9 +1,10 @@
 import HCaptchaIcon from '@/assets/logo/hcaptcha.svg'
```

Comment:
I dont understand why we have two different components for Captcha, and both are invisible, so is one of them outdated?

Reply:
oh I keep the Captcha if we need to have usual captcha integration, but I forgot to change the hcaptcha svg
should I delete it? its not used anywhere now though

Reply:
deleted the captcha modal